### PR TITLE
Feature/crud topics

### PR DESCRIPTION
**Pull request by LeandroSoria2**
### What changes does this PR include?
**Description**
Add migration with topic and test for topics
**Summary**
- Add topic model and migration
- Add test on rspec

**Preview**

image

**Trello**

(https://trello.com/c/n05HWtBl/20-2-as-an-admin-i-should-be-able-to-create-edit-delete-new-topics-for-targets)

### CODE REVIEW

File: app/models/topic.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Topic < ApplicationRecord
+  validates :name, presence: true
```
Comment (sebastiancaraballo) [inline app/models/topic.rb]: The topic name should be unique

**Review by sebastiancaraballo — COMMENTED** [review]
Remember to add Trello card to PR description

File: app/models/topic.rb (hunk 1 above)
Comment (JulianPasquale) [reply app/models/topic.rb]: Remember to add a `unique` index at db level as well 👍

**Review by AgostinaDev — APPROVED** [review]
LGTM

File: Gemfile [hunk 2]
```diff
@@ -52,6 +52,7 @@ end
 group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem 'annotate', '~> 3.2'
+  gem 'brakeman', '~> 5.2', '>= 5.2.3'
```
Comment (sebastiancaraballo) [inline Gemfile]: Wasn't this already merged? 🤔

Comment (AgostinaDev) [reply Gemfile]: @LeandroSoria2 Looks like this branch needs to be rebased

**Review by sebastiancaraballo — APPROVED** [review]

**Pull request merged**